src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -159,6 +170,26 @@ private void runGcloudCommand(List<String> args, String... topLevelCommand)
     processRunner.run(command.toArray(new String[command.size()]));
   }
 
+  private String runSynchronousGcloudCommand(List<String> args)
```

Comment:
I noticed you have Javadoc on the public methods that invoke this one, so I'm not too fussed about Javadoc here.